src/visualization.py

```diff
@@ -4,11 +4,34 @@
 import typing as T
 
 import plotly.graph_objects as go
+import plotly.express as px
+import pandas as pd
+from src.data_model import DatasetSpec, Waypoint
+from src.plan_computation import apply_trapezoidal_prof
 
-from src.data_model import Waypoint
+def plot_profiling(dist, acc, v_blur, v_max):
+    """Graphing the velocity profile for trapezoidal/triangular (time on x-axis, velocity on y-axis).
+    Args:
+        dist: distance from one image point to another
+        acc: acceleration of drone
+        v_blur: calculated drone velocity for acceptable blur
+        v_max: maximum drone speed
+    Returns:
+        Plotly figure object.
+    """
+
+    t_total, times, vels, shape = apply_trapezoidal_prof(dist, acc, v_blur, v_max)
+
+    df = pd.DataFrame({"time_s": times, "speed_mps": vels})
```

Comment:
No action required, but its not necessary to use Pandas dataframe as input for plotly. You can pass in simple arrays too.